client/src/main/java/io/pravega/client/stream/impl/TransactionInfo.java

```diff
@@ -0,0 +1,30 @@
+/**
```

Comment:
Why do we need this class? The TxnResponse message in protobuf should auto generate a corresponding class.

Reply:
Now using TxnResponse class instead of TransactionalInfo.

Reply:
@RaulGracia This comment have conflicts with Tom's review comment [ref](https://github.com/pravega/pravega/pull/6569#discussion_r817235394). After a discussion, we concluded to follow up what Tom suggested. So this particular comment becomes invalid and reverted the changes related to this comment.